.github/workflows/php.yml

```diff
@@ -0,0 +1,80 @@
+name: Latest Build
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+
+env:
+    CC_TEST_REPORTER_ID: efd18a36922628f0536f2f08cf7ceca763f5f2feb6f3638037381487ca3312ae
+    APP_ENV: development
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                operating-system: [ubuntu-latest]
+                php-versions: [ '7.2', '7.3', '7.4', '8.0' ]
```

Comment:
Can you add PHP 8.1 here to see if it passes (and maybe allow it to fail if not)?

Reply:
https://github.com/jspaetzel/micrometa/runs/6456463289?check_suite_focus=true
Looks like it works fine, LOTS of deprecation warnings, but that's to be expected.

Reply:
Deprecations are awesome, as they aren't exceptions.